k2/csrc/macros.h

```diff
@@ -21,4 +21,4 @@
 #define K2_FUNC __func__
 #endif
 
-#endif  // K2_MACROS_H_
+#endif  // K2_CSRC_MACROS_H_
```

Comment:
mm, Sorry, seems we have conflicts here as I changed it as well in https://github.com/k2-fsa/k2/pull/415. I'll merge that PR and you may need to revert your changes, many thanks.

Reply:
Conflicts have been resolved.